test/parallel_api/algorithm/alg.sorting/sort.pass.cpp

```diff
@@ -319,15 +319,14 @@ test_default_name_gen(Convert convert, size_t n)
     TestUtils::Sequence<T> in(n + 2, [=](size_t k) { return convert(k, rand() % (2 * n + 1)); });
     TestUtils::Sequence<T> expected(in);
     TestUtils::Sequence<T> tmp(in);
-    auto my_policy = oneapi::dpl::execution::make_device_policy(TestUtils::get_test_queue());
-    
+
     TestUtils::iterator_invoker<::std::random_access_iterator_tag, /*IsReverse*/ ::std::false_type>()(
-                my_policy, test_sort_op<T>(), tmp.begin(), tmp.end(), expected.begin(), expected.end(), in.begin(), in.end(),
-                    in.size(), ::std::greater<void>());
+        oneapi::dpl::execution::make_device_policy<class KernelName1>(TestUtils::get_test_queue()), test_sort_op<T>(),
+        tmp.begin(), tmp.end(), expected.begin(), expected.end(), in.begin(), in.end(), in.size(),
+        ::std::greater<void>());
     TestUtils::iterator_invoker<::std::random_access_iterator_tag, /*IsReverse*/ ::std::false_type>()(
-                my_policy, test_sort_op<T>(), tmp.begin(), tmp.end(), expected.begin(), expected.end(), in.begin(), in.end(),
-                    in.size(), ::std::less<void>());
-
+        oneapi::dpl::execution::make_device_policy<class KernelName2>(TestUtils::get_test_queue()), test_sort_op<T>(),
+        tmp.begin(), tmp.end(), expected.begin(), expected.end(), in.begin(), in.end(), in.size(), ::std::less<void>());
```

Comment:
While this makes it pass, I believe this change negates the purpose of the test.  

The purpose of the test is to check that the name generation provides unique names for radix sorts that are exactly the same other than ascending / descending sort.  This change provides the two kernels with unique policy names which will differentiate the name generation from each other on its own, no matter the handling of the ascending / descending sort status.  

The bugfix in [PR 663](https://github.com/oneapi-src/oneDPL/pull/663) disables the test for "no unnamed lambdas", and keeps its unnamed policy to properly cover this case when unnamed lambdas are allowed.